### PR TITLE
Removing initialDimensions and removeGesture methods

### PR DESCRIPTION
**Pull request by erickzanardo**
# Description

Fixes part of #623

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

If something is unclear, please submit the PR anyways and ask about what you thought was unclear.

- [x] This branch is based on the latest `master`
- [x] I have added an entry under `[next]` in `CHANGELOG.md`
- [x] I have formatted my code with `flutter format`
- [x] I have made corresponding changes to the documentation
- [x] I have added examples for new features in `doc/examples`
- [x] The continuous integration (CI) is passing

### CODE REVIEW

**Review by spydon — APPROVED** [review]
Nice!

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
  - Adding shortcut for loading Sprites and SpriteAnimation from the global cache
  - Adding loading methods for the different `ParallaxComponent` parts and refactor how the delta velocity works
  - Add tests for `Timer` and fix a bug where `progress` was not reported correctly
+ - Removing methods initialDimensions and removeGestureRecognizer to avoid confusion
```
Comment (wolfenrain) [inline CHANGELOG.md:14]: I would personally add the method names in back ticks

**Review by wolfenrain — APPROVED** [review]
LGTM, with one nit

File: CHANGELOG.md (hunk 1 above)
Comment (erickzanardo) [reply CHANGELOG.md]: Done

**Pull request merged**